### ISSUE TITLE
Make Command

### ISSUE THREAD

**Issue by lovelysandlonelys** [conrad-blucher-institute/semaphore#73]
1. Develop schedualBuilder.py (reading dspecs, init via cronjob, create message objects)
2. Make changes to dspec files to hold all the schedule infromation.
3. Develop runner.py that can parse the message object and actually run the code.

**Comment by lovelysandlonelys** [comment]
Note: Use this branch to branch from when completing tasks.

**Comment by matdenkas** [comment]
I renamed this from Orchestrator before we getting yelled at. An Orchestrator is a specific thing that basically works as a intelligent load balancer for non monolithic containerized applications. (Not super important but can explain more if you want) Son reacted to the name and I am guessing this is why. In the new Architecture diagram I called it command, we can rename it if yall come up with something better.

![Image](https://github.com/conrad-blucher-institute/semaphore/assets/64288827/96915098-60dd-4f0e-9397-248f1f27f706)
